Entity/Revision.php

```diff
@@ -1078,4 +1099,54 @@ public function hasVersionTags() : bool
     {
         return $this->contentType ? $this->contentType->hasVersionTags() : false;
     }
+
+    public function getVersionUuid(): ?UuidInterface
+    {
+        return $this->versionUuid;
+    }
+
+    public function getVersionTag(): ?string
+    {
+        return $this->rawData[Mapping::VERSION_TAG] ?? null;
+    }
+
+    public function getVersionTagDefault(): string
+    {
+        $versionTags = $this->contentType ? $this->contentType->getVersionTags() : [];
+
+        if (!isset($versionTags[0]) || !is_string($versionTags[0])) {
+            throw new \Exception(sprintf('No version tags found for contentType %s', $this->getContentTypeName()));
```

Comment:
error should say "call hasVersionTag first"

Reply:
Fixed